vtex/omnishipping
# Fix showing unavailable pickup items

#### What is the purpose of this pull request?
Fix showing unavailable pickup items

#### What problem is this solving?
Closes [thread](https://vtex.slack.com/archives/C6HS68DHD/p1579829813035200)

#### How should this be manually tested?
1. Add [items to cart](https://fernando--vtexgame1.myvtex.com/checkout/cart/add/?sku=307&qty=1&seller=1&sc=1&sku=331&qty=1&seller=1&sc=1)
4. Open the Pickup points modal and search for `Praia de botafogo 300`
3. Select `Flamengo` and the other item should be `unavailable`

#### Screenshots or example usage
n/a
#### Types of changes
